frontend/src/App.tsx

```diff
@@ -135,6 +136,7 @@ function App() {
   return (
     <ThemeProvider theme={MaterialTheme}>
       <ReactTooltip />
+      <CssBaseline />
```

Comment:
what's going on with the CSS changes in this PR? Is it related to storybook?

Reply:
yes- some of the styles were happening outside of the React framework so Storybook wasn't picking up on them and so the components looked different on storybook than the website. Since almost all of our styles were already within the React framework it seemed best to move all of them inside the framework to reduce complication and ensure the storybook renderings were the same as the actual site

Reply:
Ooh gotcha, good call!